docs/backend_standards.md

```diff
@@ -0,0 +1,200 @@
+# Backend Standards
+
+## How to create a new microservice
+
+TODO
+
+## Microservice structure
+
+```
+pkg/
+  server/
+    my-microservice/
+      controllers/
+        my_resource/
+          controller.go
+          create.go
+          list.go
+          get.go
+          update.go
+          delete.go
+      services/
+        my_resource/
+          service.go
+          create.go
+          list.go
+          get.go
+          update.go
+          delete.go
+      models/
+        my_resource/
+          model.go
+        migrate.go
+      migrations/
+        1_description.up.go
+        1_description.down.go
+        2_description.up.go
+        2_description.down.go
+      types/
+      server.go
+```
+
+### Controllers
```

Comment:
Isn't that exactly what the current `handlers` do ?

Reply:
handlers = controller, sorry maybe I should have used handlers but I was just using the language I'm used to using. Happy with either